homeassistant/components/switch/edimax.py

```diff
@@ -13,8 +13,8 @@
     CONF_HOST, CONF_NAME, CONF_PASSWORD, CONF_USERNAME)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['https://github.com/rkabadi/pyedimax/archive/'
-                '365301ce3ff26129a7910c501ead09ea625f3700.zip#pyedimax==0.1']
+REQUIREMENTS = ['https://github.com/andreipop2005/pyedimax/archive/'
```

Comment:
Home assistant requirements need to be published to pypi since a while.

Reply:
Original package maintainer did not respond to comments/messages/etc since August last year, although I really tried to make my fix pulled, but no response.

I will publish the change to pypi.

Reply:
Done.